backend/cases_import/urls.py

```diff
@@ -18,12 +18,14 @@
 api_urlpatterns = [
     path(
         route="api/case-import-action/list-create/<project>/",
-        view=views_api.CaseImportActionListCreateApiView.as_view(),
+        view=views_api.CaseImportActionViewSet.as_view({"get": "list", "post": "create"}),
```

Comment:
I don't think this endpoint is used anywhere. You should be able to generate the URLs for the view set in one line as I did for the queries etc.